.travis.yml

```diff
@@ -46,6 +52,19 @@ matrix:
     - php: 7.1
       env: CODECOVERAGE=1 DEFAULT=0
 
+  exclude:
+    - php: 5.6
+      env: DB=sqlserver db_dsn='sqlserver://SA:Strong!Passw0rd@localhost/cakephp_test?timezone=UTC'
+
+    - php: 7.0
+      env: DB=sqlserver db_dsn='sqlserver://SA:Strong!Passw0rd@localhost/cakephp_test?timezone=UTC'
+
+    - php: 7.2
+      env: DB=sqlserver db_dsn='sqlserver://SA:Strong!Passw0rd@localhost/cakephp_test?timezone=UTC'
+
+    - php: 7.4snapshot
+      env: DB=sqlserver db_dsn='sqlserver://SA:Strong!Passw0rd@localhost/cakephp_test?timezone=UTC'
```

Comment:
Instead of using `exclude` you could use `include` to add a specific build for sqlserver + 7.3

Reply:
I tried both (both are in the pull actually if you look at the first commit).

The include puts the build at the end which ends up making the full test suite run longer as sqlserver is the longest test.  This was showing how to make it run first so the total time is the same.